src/ProgramBuilder.ts

```diff
@@ -464,7 +476,7 @@ export class ProgramBuilder {
     /**
      * Parse and load the AST for every file in the project
      */
-    private async loadAllFilesAST() {
+    private async loadAllFilesAST(program: Program) {
```

Comment:
I think I'd prefer seeing the `this.program!` down below instead of making it a parameter. That changes the flow less. (people shouldn't be calling this function...but you never know)

Reply:
Fixed!